### PR TITLE
tool: Add a host side tool for Error Report

### PR DESCRIPTION
**Pull request by pillip8282**
it will parse and show log from data that TizenRT sent
it will show the command by input 'error_report --help'

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [0543bb25cf3fdf2c82624f74140d3d6066759b59] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [0543bb25cf3fdf2c82624f74140d3d6066759b59] - Code Rule Check (C++) OK.

File: tools/error_report/error_report.py [hunk 1]
```diff
@@ -0,0 +1,343 @@
+import sys
```
Comment (sunghan-chang) [inline tools/error_report/error_report.py]: Please add Samsung copyright and license sentence

**Comment by sunghan-chang** [comment]
I can not find how to use that command. Do you have any plan to show ```--help``` command or README file?

**Comment by davidfather** [comment]
I think this tool is designed to parse network-related error logs, not general error logs.
Please, specify what kind of tool is added both in title and description.

File: tools/error_report/error_report.py (hunk 1 above)
Comment (pillip8282) [reply tools/error_report/error_report.py]: Ok I'll add. I was confused because it is python.

**Comment by pillip8282** [comment]
@sunghan-chang 
Thanks for comments. We'll add the README file on error_report folder in external to let them know how to use it.
furthermore, you can check the usage with error_report.py --help

**Comment by pillip8282** [comment]
@davidfather 
Ok I'll fix it

**Comment by seinfra** [comment]
Target : [7c657cc63017b9004c59663b487ad3c8657aad1a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7c657cc63017b9004c59663b487ad3c8657aad1a] - Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@davidfather @pillip8282 As I told pillip8282, I don't know why we should say it is for network. There is no specific network functionality. It can be used for any module. Just in our case, we will use it for network. I think we don't need to make a limitation on our feature.

**Pull request closed without merging**